packages/opds-web-client/src/store.ts

```diff
@@ -23,7 +23,7 @@ export default function buildStore(initialState?: State, authPlugins?: AuthPlugi
   if (persistState) {
     composeArgs.push(persistState("preferences"));
   }
-  return createStore<State>(
+  return createStore(
```

Comment:
This was asking for 4 parameters instead of 1 with the new typings, but the documentation doesn't use any and it seems to work fine without them.